Validator/Constraints/PhoneNumber.php

```diff
@@ -39,9 +39,10 @@ class PhoneNumber extends Constraint
         self::INVALID_PHONE_NUMBER_ERROR => 'INVALID_PHONE_NUMBER_ERROR',
     ];
 
-    public $message = null;
-    public $type = self::ANY;
-    public $defaultRegion = PhoneNumberUtil::UNKNOWN_REGION;
+    public $message        = null;
+    public $type           = self::ANY;
+    public $defaultRegion  = PhoneNumberUtil::UNKNOWN_REGION;
+    public $regionProperty = null;
 
```

Comment:
shouldn't it be `regionPath` ? because we can define `sub.object.country` right?